README.mdt

```diff
@@ -0,0 +1,177 @@
+# Smartcar JS Client SDK [![Build Status][ci-image]][ci-url] [![GitHub tag][tag-image]][tag-url]
+
+The official Smartcar JavaScript SDK.
+
+## Overview
+
+The [Smartcar API](https://smartcar.com/docs) lets you read vehicle data
+(location, odometer) and send commands to vehicles (lock, unlock) using HTTP requests.
+
+To make requests to a vehicle from a web application, the end user must connect their vehicle using [Smartcar's authorization flow](https://smartcar.com/docs#authentication). The Smartcar JavaScript SDK provides an easy way to launch and handle the authorization flow to retrieve the resulting `code`.
+
+Before integrating with the JavaScript SDK, you'll need to register an application in the [Smartcar dashboard](https://dashboard.smartcar.com). Once you are registered, you will have a Client ID and Client Secret which will allow you to authorize users.
```

Comment:
this last sentence sounds kind of odd to me, but deferring to @charlottekosche here

Reply:
What about the inverting the first sentence:

>  Register an application in the [Smartcar dashboard](https://dashboard.smartcar.com) before integrating with the JavaScript SDK. Once you are registered, you will have a Client ID and Client Secret which will allow you to authorize users.

Reply:
I defer to @charlottekosche but the current version sounds better to me than the inverted version.